tests/gsrpc/extrinsics/loadTestingTool/loadTestingTool.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"avail-gsrpc-examples/internal/config"
+	"avail-gsrpc-examples/internal/extrinsics"
+	"crypto/rand"
+	"flag"
+	"log"
+	"os"
+	"sync/atomic"
+
+	gsrpc "github.com/centrifuge/go-substrate-rpc-client/v4"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/signature"
+	"github.com/centrifuge/go-substrate-rpc-client/v4/types"
+	"go.uber.org/ratelimit"
+)
+
+func main() {
+	var configJSON string
+	var config config.Config
+	flag.StringVar(&configJSON, "config", "", "config json file")
+	flag.Parse()
+
+	if configJSON == "" {
+		log.Println("No config file provided. Exiting...")
+		os.Exit(0)
+	}
+
+	err := config.GetConfig(configJSON)
+	if err != nil {
+		panic(err)
+	}
+
+	api, err := gsrpc.NewSubstrateAPI(config.ApiURL)
+	if err != nil {
+		panic(err)
+	}
+	log.Println("gsrpc connected to Substrate API...")
+
+	sub, err := api.RPC.Chain.SubscribeNewHeads()
+	if err != nil {
+		panic(err)
+	}
+	defer sub.Unsubscribe()
+	log.Println("Subscribed to new headers...")
+
+	meta, err := api.RPC.State.GetMetadataLatest()
+	if err != nil {
+		panic(err)
+	}
+
+	// if testing locally with Alice account, use signature.TestKeyringPairAlice.PublicKey as last param
+	// mneumonic for local Alice account: `bottom drive obey lake curtain smoke basket hold race lonely fit walk//Alice`
+	key, err := types.CreateStorageKey(meta, "System", "Account", signature.TestKeyringPairAlice.PublicKey)
+	if err != nil {
+		panic(err)
+	}
+
+	var accountInfo types.AccountInfo
+	ok, err := api.RPC.State.GetStorageLatest(key, &accountInfo)
```

Comment:
Got error here and it panics.

Reply:
modified some changes to take custom mnemonic. working now for me. Could you reverifiy after commit?